jobs/dynatrace-oneagent/templates/start-oneagent.sh.erb

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
+. ./service.sh
```

Comment:
Please use "source", as it makes things a bit more obvious on a quick glance :-)
```suggestion
source ./service.sh
```